app/controllers/versions_controller.rb

```diff
@@ -96,8 +77,8 @@ def which_significance_changed(cur_version_tag, prior_version_tag)
     nil
   end
 
-  # Filters
-  def create_obj
-    @object = Dor.find params[:item_id]
+  def load_and_authorize_resource
+    @cocina = Dor::Services::Client.object(params[:item_id]).find
```

Comment:
Is this always an item or can it vary? Either way, should we adopt our new naming scheme here and call this variable `@cocina_object` etc.?